server/sockets/join.js

```diff
@@ -23,17 +23,14 @@ const {
 
 const matchMaker = async (io) => {
   while (getWaitingUserLen() > 1) {
-    const [user1, user2] = getRandomPairFromWaitingList();
+    const users = getRandomPairFromWaitingList();
     
     // Check if either user is blocked
-    if (
-        await isUserBlocked(user1.loginId, user2.loginId) 
-        || 
-        await isUserBlocked(user2.loginId, user1.loginId)) {
+    if ( await isUserBlocked(users) ) {
```

Comment:
Won't this return true? Why continue if it's true

Reply:
the `continue` statement skips the rest of the loop body and moves to the next iteration. where it will try to find another pair of users instead of creating a chat for the blocked pair.

i think continue is most appropriate here, because it allows the matching process to continue with other users in the waiting list.

Reply:
if you have another way of doing same thing, please do share it

Reply:
> f creating a chat for the blocked pair.

i understand now, thanks for explaining, it's a good choice